green/process.py

```diff
@@ -65,13 +77,16 @@ class LoggingDaemonlessPool(Pool):
     """
 
     @staticmethod
-    def Process(ctx, *args, **kwds):
-        process = ctx.Process(daemon=False, *args, **kwds)
+    def Process(ctx, *args, **kwargs):
+        process = ctx.Process(daemon=False, *args, **kwargs)
         return process
 
-    def apply_async(self, func, args=(), kwds={}, callback=None, error_callback=None):
+    # FIXME: `kwargs={}` is dangerous as the empty dict is declared at import time
+    # and becomes a shared object between all instances of LoggingDaemonlessPool.
+    # In short, it is a global variable that is mutable.
+    def apply_async(self, func, args=(), kwargs={}, callback=None, error_callback=None):
```

Comment:
Oh my! Perhaps that has been the source of some of the spookiness I have encountered in the past. Nice callout.